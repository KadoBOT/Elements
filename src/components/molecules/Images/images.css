.images {
  display: flex;
  height: 600px;
  margin: 0px 35px;
}

.images > div {
  background-size: cover;
  flex: 1;
  margin: 0px 35px;
}

.images__developers {
  background: url('../../../assets/Blog/developers-analyzing.jpg')
}

.images__lab {
  background: url('../../../assets/Blog/device-lab-testing.jpg')
}
