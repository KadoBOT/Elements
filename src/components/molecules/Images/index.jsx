import React, {PropTypes} from 'react'
import R from 'ramda'

import './images.css'

const Images = ({images}) => {
  return(
    <div className="images">
      {R.map(i =>
        <div className={`images__${i}`} />
      , images)}
    </div>
  )
}

Images.propTypes = {
  props: PropTypes.type
}

export default Images
