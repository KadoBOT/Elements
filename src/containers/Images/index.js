import { connect } from 'react-redux'

import Images from '../../components/molecules/Images'

const mapStateToProps = state => ({
  images: state.images
})

export default connect(mapStateToProps)(Images)
