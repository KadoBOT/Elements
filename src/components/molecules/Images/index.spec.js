import React from 'react'
import { shallow, render, mount } from 'enzyme';

import Images from './'

it('should render the Images', () => {
    const wrapper = shallow(
        <Images images={[]} />
    );
    expect(wrapper).toMatchSnapshot();
});
