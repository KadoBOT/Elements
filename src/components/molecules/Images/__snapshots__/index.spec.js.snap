exports[`test should render the Images 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="images">
    
</div>,
  "nodes": Array [
    <div
      className="images">
      
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Images
        images={Array []} />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "images": Array [],
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="images">
          
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="images">
          
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Images
    images={Array []} />,
}
`;
